Index: Урок 1. Практическое задание/task_2.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\"\"\"\nЗадание 2.\n\nПользователь вводит время в секундах.\nПереведите время в часы, минуты и секунды и выведите в формате чч:мм:сс.\nИспользуйте форматирование строк.\n\nПример:\nВведите время в секундах: 3600\nВремя в формате ч:м:с - 1.0 : 60.0 : 3600\n\"\"\"\nsec = int(input(\"Введите время в секундах: \"))\nhours = sec / 3600\nmin = sec / 60\nprint(f\"Время в формате ч:м:с - {hours} : {min} : {sec}\")
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Урок 1. Практическое задание/task_2.py b/Урок 1. Практическое задание/task_2.py
--- a/Урок 1. Практическое задание/task_2.py	(revision dfec808b88dfdb4dc50d5859bbe3d87a422af603)
+++ b/Урок 1. Практическое задание/task_2.py	(date 1679472987650)
@@ -9,7 +9,7 @@
 Введите время в секундах: 3600
 Время в формате ч:м:с - 1.0 : 60.0 : 3600
 """
-sec = int(input("Введите время в секундах: "))
+second = int(input("Введите время в секундах: "))
 hours = sec / 3600
-min = sec / 60
+minute = sec / 60
 print(f"Время в формате ч:м:с - {hours} : {min} : {sec}")
\ No newline at end of file
